client/src/components/ElementEditor/AddElementPopoverContent.js

```diff
@@ -0,0 +1,70 @@
+/* global window */
+
+import React, { Component, PropTypes } from 'react';
+import { Button } from 'reactstrap';
+import classNames from 'classnames';
+
+/**
+ * The AddElementPopoverContent component used in the context of an ElementEditor shows the
+ * available elements that can be added to an ElementalArea.
+ */
+class AddElementPopoverContent extends Component {
+  constructor(props) {
+    super(props);
+
+    this.renderElementButtons = this.renderElementButtons.bind(this);
+    this.handleLinkClick = this.handleLinkClick.bind(this);
+  }
+
+  handleLinkClick() {
```

Comment:
This was removed right? There's still a few references to this stuff. Sad to see it go 😢

Reply:
Same here.